File: src/IrisManager.Domain/Entities/Stylist.cs
using System.ComponentModel.DataAnnotations;

namespace IrisManager.Domain.Entities
{
    public class Stylist
    {
        public int Id { get; set; }

        [Required]
        public string Name { get; set; } = string.Empty;

        [Required]
        public string Specialty { get; set; } = string.Empty;

        public bool IsActive { get; set; } = true;
        public string Email { get; set; } = string.Empty;
        public string Phone { get; set; } = string.Empty;
        public ICollection<Service> Services { get; set; } = new List<Service>();
    }
}

File: src/IrisManager.Client/src/app/core/models/stylist.ts
export interface Stylist {
  id: number;
  name: string;
  email: string;
  phone: string;
  isActive: boolean;
  specialty?: string;
  fullSpecialties?: string;
  serviceIds?: number[];
  serviceNames?: string[];
}

export interface StylistCreateDTO {
  name: string;
  email: string;
  phone: string;
  isActive: boolean;
  serviceIds?: number[];
}

export interface StylistUpdateDTO {
  id: number;
  name: string;
  email: string;
  phone: string;
  isActive: boolean;
  serviceIds?: number[];
}